<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartforph</class>
 <widget class="QWidget" name="chartforph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>10</y>
     <width>871</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Chart:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="chartComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="timeComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="themeLabel">
      <property name="text">
       <string>Theme</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="themeComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="animatedLabel">
      <property name="text">
       <string>Animation:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="animatedComboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="legendLabel">
      <property name="text">
       <string>Legend:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="legendComboBox"/>
    </item>
    <item>
     <widget class="QCheckBox" name="antialiasCheckBox">
      <property name="text">
       <string>Anti-aliasing</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>921</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
